template/config/cypress/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
-    "test:e2e": "start-server-and-test preview http://localhost:4173/ 'cypress open --e2e'",
-    "test:e2e:ci": "start-server-and-test preview http://localhost:4173/ 'cypress run --e2e'"
+    "test:e2e": "start-server-and-test preview :4173 'cypress run --e2e'",
+    "test:e2e:dev": "start-server-and-test 'vite dev --port 4173' :4173 'cypress open --e2e'"
```

Comment:
I chose to force the dev server to run on 4173 instead of 5173 as in the playwright PR https://github.com/vuejs/create-vue/pull/182 because `start-server-and-test` doesn't have an option like `reuseExistingServer`, therefore has a higher chance of port collisions during dev.